components/GlobalStyles.js

```diff
@@ -19,7 +19,7 @@ const GlobalStyles = createGlobalStyle`
 
     @font-face {
         font-family: AppleGothic;
```

Comment:
das müsstest du irgendwann noch anpassen - im Zuge dessen würde ich auch die Schriftart in den Global Styles definieren :)
(mit cmd+shift+f kannst du in allen Dateien suchen)